docker/tensorflow-application/analysis/startScript.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Start the first process
+python analysis.py &
```

Comment:
This isn't necessary (or even correct). In this case you don't even need an separate script to start your process. Just change your Dockerfile to have `ENTRYPOINT ["python", "analysis.py"]`.